app/views/editor/question.html.erb

```diff
@@ -108,8 +108,7 @@
           <% end %>
 
           <a id="publish" tabindex="5" class="publish btn btn-primary btn-lg">Publish</a>
-          <a tabindex="6" data-previewing-text="Previewing (click to edit)" class="btn btn-outline-secondary btn-lg preview-btn" onClick="$E.toggle_preview()">Preview</a>
-
+          <a tabindex="6" id="toggle-preview-button-main" data-previewing-text="<%= t('wiki.edit.previewing_text') %>" class="btn btn-outline-secondary btn-lg preview-btn" onClick="$E.toggle_preview()"><%= t('wiki.edit.preview') %></a>
```

Comment:
I wonder if this phrase could be different. Like, `Hide Preview` (that's how it is on comments) instead of `Previewing (click to edit)`. The latter is kind of confusing.

Reply:
@noi5e `Hide Preview` is used in some parts in comment preview I guess, but as initial code had `Previewing (click to edit)` I didn't change it

Reply:
@imajit, Thanks, yes, I know you didn't write that piece of text, but it's worth saying :)